ui/src/components/AppDetailToolbar.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { Breadcrumb, BreadcrumbItem, Toolbar, ToolbarGroup, Button, ToolbarItem } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import './AppDetailedToolbar.css';
+
+class AppDetailToolbar extends React.Component {
+  componentWillMount () {
+    this.setState({
+      redirect: false
+    });
+  }
+
+  onDisableButton = () => {
+    console.log('Disable button clicked');
+  }
+
+  onClick = () => {
+    this.setState({
```

Comment:
This will need to open up the modal if the current form state is dirty.

Reply:
@davidffrench yes, this is a temporary usage until we have the the redux state implementation complete. @damienomurchu will be making some changes and @craicoverflow will be adding some changes as part of his pr. Once those are in I will be able to update this correctly.

Reply:
perfect